ors-engine/src/main/java/org/heigit/ors/routing/pathprocessors/ExtraInfoProcessor.java

```diff
@@ -235,9 +235,11 @@ public class ExtraInfoProcessor implements PathProcessor {
                 }
             }
 
+            // TODO: there are too many things identifying this information:
```

Comment:
Agreed! 👍

You might want to use the opportunity now to clean this up a bit. Do you already have a specific solution in mind?